src/uid2DiiNormalization.ts

```diff
@@ -0,0 +1,78 @@
+enum EmailParsingState {
+  Starting,
+  SubDomain,
+}
+
+export type NormalizedPhone = string;
```

Comment:
could you please explain the use of this rather than just using `string`? do plan to change this type in the future?